lib/mocha/ruby_version.rb

```diff
@@ -1,11 +1,4 @@
 require 'mocha/deprecation'
 
 module Mocha
-  RUBY_V2_PLUS = Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('2')
-
-  unless RUBY_V2_PLUS
-    Mocha::Deprecation.warning(
-      'Versions of Ruby earlier than v2.0 will not be supported in future versions of Mocha.'
-    )
-  end
```

Comment:
Kept this file as I expect to use it in https://github.com/freerange/mocha/pull/554

Reply:
👍